zubhub_frontend/zubhub/src/components/social_share_buttons/socialShareButtons.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { ButtonGroup } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import LinkIcon from '@material-ui/icons/Link';
+import WhatsAppIcon from '@material-ui/icons/WhatsApp';
+import FacebookIcon from '@material-ui/icons/Facebook';
+import IconButton from '@material-ui/core/IconButton';
+import { useMediaQuery } from '@material-ui/core';
+
+const styles = makeStyles({
```

Comment:
style should be moved to the styles folder, just like is the case for other components